public/employers/american-airlines.yml

```diff
@@ -51,3 +51,10 @@ citations:
     sources:
       - source: CNBC
         link: https://web.archive.org/web/20200412193826/https://www.cnbc.com/2020/03/13/workforce-wire-coronavirus-heres-what-every-major-company-is-doing-about-the-pandemic.html
+  - summary: After receiving a bailout from the government under the stipulation that they not lay off any workers through September 30; on September 30, they announced that they would furlough 19,000 workers starting October 1 unless they received more stimulus money.
```

Comment:
I think this could be worded better:
"After receiving a bailout from the government under the stipulation that they not lay off any workers through September 30; the company announced on that day that they would furlough 19,000 workers starting October 1 unless they received more stimulus money."